source/tutorials/write-update-documents.txt

```diff
@@ -0,0 +1,291 @@
+
+To update multiple documents from a MongoDB collection, call the ``updateMany()``
+method and pass in your query filter. Then, pass in your
+update operators as the second argument to the ``updateMany()`` method. After you call the
+``updateMany()`` method, chain the ``subscribe()`` method and pass in a subscriber.
+
+The following example updates all documents in the ``restaurants`` collection
+with a ``borough`` value of ``"Brooklyn"`` to have a star value of 4:
+
+.. code-block:: java
+   :copyable: true
+
+    ObservableSubscriber<UpdateResult> updateManySubscriber = new SubscriberHelpers.OperationSubscriber<>();
+    restaurants.updateMany(eq("cuisine", "Pizza"), set("cuisine", "Pasta")).subscribe(updateManySubscriber);
+    updateManySubscriber.await();
+
+Customize the Update Operation
+------------------------------
+
+The ``UpdateOptions`` class contains methods that modify
+the behavior of update methods. To use the ``UpdateOptions``
+class, construct a new instance of the class, then call one or more of its methods
+to modify the update operation. You can chain these method calls together.
+To modify the behavior of the update operation, pass the class instance and
+chained method calls as the second argument to the
+``updateOne()`` or ``updateMany()`` method.
+
+You can use the following methods in the ``updateOptions`` class
+to modify an update method. All methods are optional.
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``array_filters (List<? extends Bson> arrayFilters)``
+     - | A list of filters that specifies which array elements an update applies
+         to.
+
+   * - ``bypass_document_validation (Boolean bypass_document_validation)``
+     - | Specifies whether the update operation bypasses document validation. This lets you 
+         update documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+      
+   * - ``collation (Collation collation)``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the MongoDB Server manual.  
+
+   * - ``comment (Bson comment)``
+     - | A comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         MongoDB Server manual.  
+
+   * - ``comment (String comment)``
+     - | A comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         MongoDB Server manual.  
+
+   * - ``hint (Bson hint)``
+     - | Gets or sets the index to scan for documents. 
```

Comment:
I: In the [API docs](https://mongodb.github.io/mongo-java-driver/5.1/apidocs/mongodb-driver-core/com/mongodb/client/model/UpdateOptions.html), it looks like hint() is just the setter method and getHint() would be the getter. So I think you can remove the "Gets or" part of the sentence.

Also, I think differentiating this method from the next one in the table (hint(String hint)) would be helpful:

```suggestion
     - | Sets the index for the operation as a ``Bson`` value.
```